hello_world/README.md

```diff
@@ -0,0 +1,146 @@
+# Hello World
+
+The simplest yet completely legitimate SCORE that returns "Hello" in the response message. 
+
+```python
+from iconservice import *
+
+class HelloWorld(IconScoreBase):
+
+    def __init__(self, db: IconScoreDatabase) -> None:
+        super().__init__(db)
+
+    def on_install(self) -> None:
+        super().on_install()
+
+    def on_update(self) -> None:
+        super().on_update()
+
+    @external(readonly=True)
+    def hello(self) -> str:
+        return "Hello"
+```
+
+ 
+
+We will add a little tweak to our HelloWorld.  
+
+- Give it a name, "HelloWorld"
+- Print logs. 
+- Allow it to accept ICX and other tokens.
+
+```python
+from iconservice import *
+
+TAG = 'HelloWorld'
+
+class HelloWorld(IconScoreBase):
+
+    def __init__(self, db: IconScoreDatabase) -> None:
+        super().__init__(db)
+
+    def on_install(self) -> None:
+        super().on_install()
+
+    def on_update(self) -> None:
+        super().on_update()
+    
+    @external(readonly=True)
+    def name(self) -> str:
+        return "HelloWorld"
+
+    @external(readonly=True)
+    def hello(self) -> str:
+        Logger.info('Hello, world!', TAG)
+        return "Hello"
+
+    @payable
+    def fallback(self):
+        Logger.info('fallback is called', TAG)
+
+    @external
+    def tokenFallback(self, _from: Address, _value: int, _data: bytes):
+        Logger.info('tokenFallabck is called', TAG)
+```
+
+`fallback` function is added to accept ICX. `fallback` function is executed when the contract receives a transaction request without `data` part. Not having `data` in the transaction means no method name is specified. In such case, `fallback` function is invoked if the function is provided. If `fallback` function is not given, the transaction will fail. 
+
+ICX transfer request message does not have `data` part. So, the default behavior of any contract is rejecting incoming ICX. This design is to prevent any accidental ICX transfer to a contract. 
+
+Only functions with `@payable` decorator are permitted to receive ICX coins, therefore, our `fallback` function should be also  `@payable` because we want the contract receive ICX.  You don't need to implement anything in the fallback function to handle the ICX transfer. Having payable fallback function declares that the contract is designed to accept ICX coin transfer. 
```

Comment:
```
--- we want the contract receive ICX
+++ we want the contract to receive ICX
```


```
--- You don't need to implement anything in the fallback function to handle the ICX transfer. Having payable fallback function declares that the contract is designed to accept ICX coin transfer. 
+++ By design, `@payable` functions handle ICX coin transfers, so you don't need to implement anything regarding this matter for these functions.

Reply:
fixed the type. 
for the "@payable", it is not exactly the payable function who handles the coin transfer.  iconservice handles the coin transfer, existence of payable fallback function confirms the contract is valid recipient.